.github/workflows/test_template.yml

```
@@ -0,0 +1,42 @@
+name: Test Template
+
+on: push
+
+env:
+  RUBY_VERSION: 3.0.1
```

How about using `RUBY_VERSION` env also here?
```suggestion
  RUBY_VERSION: ${{ env.RUBY_VERSION }}
```